dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/LogProviderConfigImpl.java

```diff
@@ -578,7 +608,10 @@ private static enum InitConfgAttribute {
         ROLLOVER_INTERVAL("rolloverInterval", "com.ibm.ws.logging.rollover.interval"),
         MAX_FFDC_AGE("maxFfdcAge", "com.ibm.ws.logging.max.ffdc.age"),
         FFDC_CLEANUP_START_DELAY("ffdcCleanupStartDelay", "com.ibm.ws.logging.ffdc.cleanup.start.delay"),
-        NEW_LOGS_ON_START("newLogsOnStart", FileLogHolder.NEW_LOGS_ON_START_PROPERTY);
+        NEW_LOGS_ON_START("newLogsOnStart", FileLogHolder.NEW_LOGS_ON_START_PROPERTY),
+        THROTTLE_MAX_MESSAGES_PER_WINDOW("throttleMaxMessagesPerWindow", "com.ibm.ws.logging.throttle.max.messages.per.window"),
+        THROTTLE_TYPE("throttleType", "com.ibm.ws.logging.throttle.type"),
+        THROTTLE_MAP_SIZE("throttleMapSize", "com.ibm.ws.logging.throttle.map.size");
```

Comment:
The bootstrap property is not set in the meta type. Refer to the meta type.xml for concerns regarding the internal throttle map size attribute.